src/app/common/HorizontalDefinitionCard.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div class="definition-card">
+    <div class="definition-card-title">
+      <slot name="icon" />
+
+      <slot name="title" />
+    </div>
+
+    <div class="definition-card-container">
+      <slot name="body" />
+    </div>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.definition-card {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  gap: $kui-space-40;
+}
+
+.definition-card-title {
+  display: flex;
+  align-items: flex-end;
+  gap: $kui-space-40;
+}
+
+.definition-card-container {
+  display: flex;
+  align-items: flex-start;
+  font-weight: $kui-font-weight-bold;
+}
+</style>
+
+<style lang="scss">
+.definition-card-container > * {
+  min-width: 0;
+}
+</style>
```

Comment:
Rather than copy/paste the code from `DefinitionCard` and tweak a line or two, is there anything else we can do here. I know sometimes the copy/paste thing is ok, just it feels like in this case there could be a better way to do this.

I think at the very least we should use different classnames, extra "at the very least" for the non-scoped style. But let me know if you have any ideas for a better way to do this.

Reply:
I decided to integrate this into DefinitionCard via an optional `layout` prop. Initially I thought there’d be more differences, but that didn’t turn out that way.